README.md

```diff
@@ -18,9 +16,9 @@ While it is easier to pull from DockerHub, the image will default to the softwar
 
 | Software    | Version       |
 |-------------|---------------|
-| [Accumulo]  | 2.0.0-alpha-1 |
-| [Hadoop]    | 3.1.1         |
-| [Zookeeper] | 3.4.13        |
+| [Accumulo]  | 2.0.0         |
+| [Hadoop]    | 3.2.1         |
+| [Zookeeper] | 3.6.0         |
```

Comment:
The ZooKeeper trademark has an upper-case `Z` and `K`. We make that mistake in a lot of places, but I try to call it out whenever I see it in a new code change.

```suggestion
| [ZooKeeper] | 3.6.0         |
```

The corresponding markdown link label would also need to change.

Reply:
I have addressed this in the new commit. Thanks